app/mailers/snack_mailer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class SnackMailer < ApplicationMailer
+  def self.snack_creation(snack)
+    admins = User.admins
+    mail_list = admins.map(&:email)
```

Comment:
You can query only for the email column and get an array of emails by using:

```suggestion
    mail_list = User.admins.pluck(:email)
```